README.md

```diff
@@ -2,6 +2,18 @@
 
 Utilities for Redis functionality
 
+# Publishling to npm
+
+To update the module on npm:
+
+1. Ensure `package.json`'s version number has been incremented.
+2. Checkout the main branch
+3. Pull and update the main branch
+4. `npm run publish` to publish the package to npm.
```

Comment:
what are the permissions to publish? Only you can publish or anyone can publish?

Reply:
You're npm account has admin permissions to our organization. I assume you would be able to make necessary changes.

![image](https://github.com/KGS-Ben/redis-utils/assets/88667020/f73002b7-4898-4b02-b90d-0fb727632394)